tests/testthat/test-data.R

```diff
@@ -1,71 +1,112 @@
-testthat::context("Data input")
-testthat::test_that("Creating a WTSS coverage", {
-    testthat::skip_on_cran()
+context("Data input")
+test_that("Creating a WTSS coverage", {
+    #skip_on_cran()
```

Comment:
All `skip_on_cran()` calls were commented. Some of these tests depends on third-party services (e.g. SATVEG, WTSS) that can be unavailable for some reason. Its is important to avoid this dependency in CRAN.

Reply:
The issue related to test coverage will not be closed by this PR. I will review this before closing the issue.